packages/v4/src/content/design-guidelines/components/drawer/drawer.md

```diff
@@ -12,4 +12,11 @@ Drawers appear as a sliding panel that can be attached to the bottom or right ed
 
 In addition to primary-details, the drawer component is frequently used in [notification drawers](/components/notification-drawer) or terminal windows.
 
+### Splitter in a drawer
 
+A **splitter** allows you to create a layout with resizable panes. The orientation of a splitter can be set to **vertical** or **horizontal**.
+
+### Usage
+Add a splitter to drawer if you need to resize the width or height of a panel and give more space to a given content. A splitter resizes the width or height of a panel depending on the orientation of a splitter. If data shown in a drawer has enough space, then you don’t need to use a splitter.
```

Comment:
Can you reword this first sentence to say:
"Add a splitter to a drawer if you need to resize the width or height of a panel to give content more space." and then delete the second line, because I feel like you already said that in the first!

So delete "A splitter resizes the width or height of a panel depending on the orientation of a splitter"